<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>160</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="fbiBtn">
      <property name="text">
       <string>FBI</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ciaBtn">
      <property name="text">
       <string>CIA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deaBtn">
      <property name="text">
       <string>DEA</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
